lib/generated/api.d.ts

```diff
@@ -0,0 +1,371 @@
+import http = require("http");
```

Comment:
by `api.d.ts` you meant all the different methods I placed in  this file right? I was not 100% sure

Reply:
Yes